### PR TITLE
added docker setup for local 11g database

### PR DESCRIPTION
**Pull request by mbocek**
Setup for running IT tests in travis

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,21 @@
...
 jdk:
   - oraclejdk8
+
+services:
+  - docker
+
+before_install:
+- docker pull sath89/oracle-xe-11g
+- docker run -d --name dbadvisor-test --shm-size=1G -p 8080:8080 -p 1521:1521 -v `pwd`/dbadvisor-core/src/test/resources/scripts:/docker-entrypoint-initdb.d sath89/oracle-xe-11g
+- while true; do if [ $(docker inspect --format="{{.Id}}" dbadvisor-test | xargs docker logs | grep "Database ready to use. Enjoy! ;)" | wc -l) -gt 0 ]; then echo "ready"; break; fi; echo "waiting ..."; sleep 10; done
```
Comment (thradec) [inline .travis.yml]: I just found `testcontainers`, see https://www.testcontainers.org/usage/database_containers.html wdyt?

File: pom.xml [hunk 2]
```diff
@@ -140,4 +154,16 @@
...
+            <id>maven.oracle.com</id>
+            <name>oracle-maven-repo</name>
+            <url>https://maven.oracle.com</url>
+            <layout>default</layout>
+            <releases>
+                <enabled>true</enabled>
+                <updatePolicy>always</updatePolicy>
+            </releases>
+        </repository>
+    </repositories>
```
Comment (thradec) [inline pom.xml]: couldn't we use this one? without the OTN authentication

https://maven.atlassian.com/3rdparty/com/oracle/ojdbc6/11.2.0.4.0-atlassian-hosted/

```
   <repositories>
        <repository>
            <id>atlassian-3rd-party</id>
            <url>https://maven.atlassian.com/3rdparty/</url>
        </repository>
    </repositories>

        <dependency>
            <groupId>com.oracle</groupId>
            <artifactId>ojdbc6</artifactId>
            <version>12.1.0.1-atlassian-hosted</version>
            <scope>test</scope>
        </dependency>
```

File: .travis.yml (hunk 1 above)
Comment (mbocek) [reply .travis.yml]: I have played with this piece of technology. Unfortunately, the start of the container is time-consuming and i have decided to use "standalone" DB.

File: dbadvisor-core/src/test/java/com/github/embeditcz/dbadvisor/core/AbstractIT.java [hunk 3]
```diff
@@ -9,22 +9,27 @@
...
+
         @Bean
         public HikariDataSource dataSource() {
             HikariDataSource ds = new HikariDataSource();
-            ds.setJdbcUrl("jdbc:oracle:thin:@//dockerip:1531/ORCL"); // TODO tmp
-            ds.setUsername("SAA");
-            ds.setPassword("p1");
+            ds.setJdbcUrl(format("jdbc:oracle:thin:@//%s:%s/XE", HOST, PORT)); // TODO tmp
+            ds.setUsername("bank");
+            ds.setPassword("bank");
```
[the remark this answers is not in the document]
Comment (mbocek) [reply dbadvisor-core/src/test/java/com/github/embeditcz/dbadvisor/core/AbstractIT.java]: Let's do that in next PR. But good point.

File: dbadvisor-core/src/test/resources/logback-test.xml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    </appender>
+
+    <root level="info">
+        <appender-ref ref="STDOUT"/>
+    </root>
+
+    <logger name="org.testcontainers" level="INFO"/>
+    <logger name="org.apache.http" level="WARN"/>
+    <logger name="com.github.dockerjava" level="WARN"/>
+    <logger name="org.zeroturnaround.exec" level="WARN"/>
```
[the remark this answers is not in the document]
Comment (mbocek) [reply dbadvisor-core/src/test/resources/logback-test.xml]: removed

File: pom.xml (hunk 2 above)
Comment (mbocek) [reply pom.xml]: done

**Pull request merged**